packages/matter-node.js/src/matter/interaction/InteractionClient.ts

```diff
@@ -175,9 +175,12 @@ export class InteractionClient {
 
     async invoke<C extends Command<any, any>>(endpointId: number, clusterId: number, request: RequestType<C>, id: number, requestSchema: TlvSchema<RequestType<C>>, _responseId: number, responseSchema: TlvSchema<ResponseType<C>>, optional: boolean): Promise<ResponseType<C>> {
         return this.withMessenger<ResponseType<C>>(async messenger => {
+            // if the invoke call do not have arguments we want to send an empty tlv object to not suppress the args field (even if MAY be allowed by specs)
+            const args = (requestSchema === TlvNoArguments && request === undefined) ? TlvObject({}).encodeTlv({}) : requestSchema.encodeTlv(request);
```

Comment:
TlvSchema are made to encapsulate encoding / decoding so TlvNoArguments should define its own schema instead of adding complexity here:
```
const TlvEmptyObject = TlvObject({});

export class TlvNoArguments extends TlvSchema<void> {

    override encodeTlvInternal(writer: TlvWriter, value: void, tag: TlvTag = {}): void {
        if (value !== undefined) throw new Error("No value should be passed");
        TlvEmptyObject.encodeTlvInternal(writer, {}, tag);
    }

  [...]

}
```

Reply:
done